src/Symfony/Component/Cache/Tests/DependencyInjection/CachePoolPassTest.php

```diff
@@ -214,4 +215,27 @@ public function testChainAdapterPool()
         $this->assertInstanceOf(ChildDefinition::class, $doctrineCachePool);
         $this->assertSame('cache.app', $doctrineCachePool->getParent());
     }
+
+    public function testChainInChainException()
+    {
+        $this->expectException(InvalidArgumentException::class);
```

Comment:
I would move this direct before the process() call

Reply:
Fixed in commit 97c8dde